chall01/kmoilane.py

```diff
@@ -0,0 +1,37 @@
+MORSE_DICTIONARY = { 'A':'.-', 'B':'-...',
+                    'C':'-.-.', 'D':'-..', 'E':'.',
+                    'F':'..-.', 'G':'--.', 'H':'....',
+                    'I':'..', 'J':'.---', 'K':'-.-',
+                    'L':'.-..', 'M':'--', 'N':'-.',
+                    'O':'---', 'P':'.--.', 'Q':'--.-',
+                    'R':'.-.', 'S':'...', 'T':'-',
+                    'U':'..-', 'V':'...-', 'W':'.--',
+                    'X':'-..-', 'Y':'-.--', 'Z':'--..',
+                    'a':'.-', 'b':'-...',
+                    'c':'-.-.', 'd':'-..', 'e':'.',
+                    'f':'..-.', 'g':'--.', 'h':'....',
+                    'i':'..', 'j':'.---', 'k':'-.-',
+                    'l':'.-..', 'm':'--', 'n':'-.',
+                    'o':'---', 'p':'.--.', 'q':'--.-',
+                    'r':'.-.', 's':'...', 't':'-',
+                    'u':'..-', 'v':'...-', 'w':'.--',
+                    'x':'-..-', 'y':'-.--', 'z':'--..'}
+
+import sys
+
+message = ''
+result = ''
+
+if len(sys.argv) != 2 :
+    print('usage: ./xlogin.py <a-zA-Z string>')
```

Comment:
kmoilane.py not xlogin.py

I would also recommend using sys.argv[0] instead of hardcoding the name app.